schema_cache.rst

```diff
@@ -0,0 +1,131 @@
+.. _schema_cache:
+
+Schema Cache
+============
+
+Certain PostgREST features need information from the database schema; however, accessing the database directly would be too costly, that is why PostgREST generates a database schema cache on startup and gets the information from it instead:
+
++--------------------------------------------+-------------------------------------------------------------------------------+
+| Feature                                    | Required Metadata                                                             |
++============================================+===============================================================================+
+| :ref:`resource_embedding`                  | Foreign key constraints (to determine relationships between tables)           |
++--------------------------------------------+-------------------------------------------------------------------------------+
+| :ref:`Database Functions <s_procs>`        | Function's metadata (parameters, return type, volatility and                  |
+|                                            | `overloading <https://www.postgresql.org/docs/current/xfunc-overload.html>`_) |
++--------------------------------------------+-------------------------------------------------------------------------------+
+| :ref:`Upserts <upsert>`                    | Primary keys                                                                  |
++--------------------------------------------+-------------------------------------------------------------------------------+
+| :ref:`insert_update`                       | Primary keys (in order to return the Location header)                         |
```

Comment:
Only insertions need the pk.
```suggestion
| :ref:`Insertions <insert_update>`                       | Primary keys (in order to return the Location header)                         |
```

Reply:
@laurence Btw, the Location header is optional now(`return=headers`). Perhaps this need to be cleared here.

Reply:
@laurenceisla Btw, the Location header is optional now(return=headers). Perhaps we should clear this here.